frontend/elements/src/entry/home/widgets/popup-section.ts

```diff
@@ -0,0 +1,76 @@
+import { MEDIA_UI } from '@utils/path';
+ import { LitElement, html, css, customElement, property } from 'lit-element';
+ 
+ export type Image = "" | "play"| "askforhelp" |"tryagain" ;
+
+
+  @customElement('popup-section')
+ export class _ extends LitElement {
+  static get styles() {
+    return [css`
+    .inside-wrapper{
+       display:flex;
+       margin-left:170px;
+       margin-top:130px;
+    }
+   main{
+    width: 760px;
+    height: 462px;
+    position:relative;
+  }
+    
+  
+  ::slotted([slot=title]){
+    position:relative;
+    text-align: center;
+    top:86px;
+
+
+  }
+  
+  .img{
+    bottom:-5px;
+    left:0px;
+     position:absolute;
+  }
+ 
+  ::slotted([slot=baloon]){
+    width: 216.6px;
+    height: 143.1px;
+     bottom:40px;
+     left:120px;
+      position:absolute;
+  }
+  ::slotted([slot=square]){
+     margin-left:32px;
+     display:block;
+  }
+
+    `];
+  }
+  @property()
+  kindimage:Image = ""; 
+
+
+  render() {
+    const {kindimage } = this;
+
+    const path = kindimage === "play" ? " Illustration_JIG_Sad_1.png"
```

Comment:
if empty path is a legitimate option, hide the `img-ui` completely in that case instead of setting the path to `""`